src/github.com/getlantern/connpool/connpool.go

```diff
@@ -9,9 +9,7 @@ import (
 )
 
 const (
-	DefaultClaimTimeout         = 10 * time.Minute
-	DefaultRedialDelayIncrement = 50 * time.Millisecond
-	DefaultMaxRedialDelay       = 1 * time.Second
+	DefaultClaimTimeout = 10 * time.Minute
```

Comment:
I looked for these exported constants `DefaultClaimTimeout`, `DefaultRedialDelayIncrement`, `DefaultMaxRedialDelay` and didn't find any piece of code that's actually using them besides the `connpool` package. Also, they're only used once (within the `New()` function). Should we keep them exported?